programmers/K번째수.js

```diff
@@ -0,0 +1,16 @@
+function solution(array, commands) {
+    
+    let answer = [];
+    let new_arr = [];
```

Comment:
```suggestion
    let targets = [];
```
javscript는 카멜케이스를 사용하는데요 그래서 new_arr가 아니라 newArr이런식으로 사용해야합니다.
그리고 newArr같은 이름보다 targets이나 temp같은 좀 더 명확한 뜻이 있는 변수명을 사용하는 것이 좋을 것 같아요